obspy/imaging/cm.py

```diff
@@ -24,42 +31,69 @@
 import numpy as np
 from matplotlib.cm import get_cmap
 from matplotlib.colors import LinearSegmentedColormap
+from matplotlib.colors import ListedColormap
 
 
-def _get_cmap(name, lut=None, reverse=False):
+def _get_cmap(file_name, lut=None, reverse=False):
```

Comment:
Any specific reason for the unrelated arg name change `name` -> `file_name`? Better avoid unnecessary API changes, even if it's a private function..